modules/cluster-limitations-local-zone.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
```

Comment:
@dfitzmau I think you need to delete this module as it has been moved.

Reply:
Hi @mtulio . Well spotted. I removed this module.